main.tf

```diff
@@ -1,13 +1,77 @@
-resource "random_integer" "example" {
+#-----------------------------------------------------------------------------------------------------------------------
+# Subscribe the Acccount to Security Hub
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_account" "this" {
   count = module.this.enabled ? 1 : 0
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally subscribe to Security Hub Standards 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards.html
+#-----------------------------------------------------------------------------------------------------------------------
+resource "aws_securityhub_standards_subscription" "this" {
+  for_each      = local.enabled_standards_arns
+  depends_on    = [aws_securityhub_account.this]
+  standards_arn = each.key
+}
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Optionally configure Event Bridge Rules and SNS subscriptions 
+# https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cwe-integration-types.html
+#-----------------------------------------------------------------------------------------------------------------------
+module "sns_topic" {
+  source = "git::https://github.com/cloudposse/terraform-aws-sns-topic.git?ref=tags/0.9.0"
+  count  = local.create_sns_topic ? 1 : 0
+
+  attributes      = ["securityhub-sns-topic"]
```

Comment:
```suggestion
  attributes      = ["securityhub"]
```

1) to make it shorter
2) we usually don't include resource types in resource names (there are some exceptions though)